config/locales/views/pages/en.yml

```diff
@@ -1,13 +1,81 @@
+        see_how_it_works: 'See how it works'
+        sign_in_to_start: 'Sign in and get started'
+      why_classroom_section:
+        heading: 'Why try Classroom?'
+        items:
+        - heading: 'Spend more time with students, less on setup.'
+          content: 'Students accept an assignment with one link, so you can get straight to the material.'
+        - heading: 'Bootstrap group assignments in a snap.'
+          content: 'Invite students to a shared repository, and cap the number of students per group. Use the same groups over and over again, or create new ones.'
+        - heading: 'Download all repositories with the click of a button.'
+          content: 'When you''re ready to grade, download assignments to your machine from GitHub Classroom, or use our new desktop client, Classroom Assistant.'
+        - heading: 'More insight into student work than ever before.'
+          content: 'See when students accept the assignment, and access their work from the moment they start. With version control, catch when they get stuck and help them rewind.'
+        - heading: 'You are in control.'
+          content: 'Students can work individually or in groups, in public or in private. Set permissions for teaching assistants or graders.'
+        - heading: 'Scales for large courses with ease.'
+          content: 'If you have a small course, Classroom will make your life easier and save you time. If you have hundreds of students, we have your covered: as many repositories as you need, and webhooks to integrate automated testing tools.'
+        - heading: 'Works with your Learning Management System (LMS).'
+          content: 'Students submit a link to their assignment repository to your learning management system. Give feedback through comments in GitHub, but keep grades in your LMS.'
+        - heading: 'Are you super-advanced?'
+          content: Do you want to build your own tools? GitHub Classroom is [open source](https://github.com/education/classroom), and we 💖 contributions.
+      what_teachers_say_section:
+        heading: 'What teachers are saying'
+        items:
+        - quote: 'GitHub Classroom takes the intimidation out of using GitHub for noobs.'
+          name: 'Paul Intrevado'
+          school: 'University of San Francisco'
+        - quote: 'Excellent solution and with the power of Git and GitHub, mind blowing!'
+          name: 'Dr Arindam Basu'
+          school: 'University of Canterbury'
+        - quote: 'This system organizes the resources required to teach coding in a way that significantly simplifies the task of assigning and collecting work from students.'
+          name: 'Stephen Cerruti'
+          school: 'e3 Civic High School'
+        - quote: 'After today I realize how simple this process really is. I never realized it would be so quick to set all of these things up and start using them for students. On top of this the support community is absolutely amazing!!'
```

Comment:
add link to community in 'support community' https://education.github.community/